docs/installation.md

```diff
@@ -114,7 +99,7 @@ Create a Service with type `LoadBalancer` using the appropriate manifest for you
 - For GCP or Azure:
 
    ```shell
-   kubectl apply -f deploy/manifests/service/loadbalancer.yaml
+   kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.0.0/deploy/manifests/service/loadbalancer.yaml
```

Comment:
Do we have an artifact for the aws loadbalancer on line 117?

Reply:
Good catch